File: src/Git.Domain/Models/Casualty.cs
using Newtonsoft.Json;

namespace Git.Domain.Models
{
    /// <example>
    ///      {
    ///        "age": 0,
    ///        "class": "string",
    ///        "severity": "string",
    ///        "mode": "string",
    ///        "ageBand": "string"
    ///      }
    /// </example>>
    public class Casualty
    {
        public int Age { get; set; }

        public string Class { get; set; }

        public Severity Severity { get; set; }

        public string Mode { get; set; }

        public string AgeBand { get; set; }

    }
}

File: src/node-accident-statistics/src/models/casualty.ts
export type SeverityOptionValues = 0 | 1 | 2;

// tslint:disable-next-line:interface-name
export interface Casualty {
    age?: number;
    class?: string;
    severity?: SeverityOptionValues;
    mode?: string;
    ageBand?: string;
    $type: string | undefined;
}
